spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-sample/src/main/resources/application.properties

```diff
@@ -1 +1,5 @@
+# Fill out the following properties
+spring.cloud.gcp.project-id=
+spring.cloud.gcp.credentials.location=
+spring.cloud.gcp.pubsub.universeDomain=
```

Comment:
See existing pattern for how to request users provide their own value:
https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/main/spring-cloud-gcp-samples/spring-cloud-gcp-sql-mysql-sample/src/main/resources/application.properties#L1-L2

Then how to override for testing:
https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/main/spring-cloud-gcp-samples/pom.xml#L177